whois-commons/src/main/java/net/ripe/db/whois/common/rpki/RpkiService.java

```diff
@@ -8,6 +8,7 @@
 import net.ripe.db.whois.common.ip.Ipv6Resource;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
 import org.springframework.stereotype.Service;
```

Comment:
Why this doesnot show @Component on class level?

Reply:
It is not longer there....you need to reload the page probably?